online-tutorial-series/01_xarray_fundamentals.ipynb

```diff
@@ -0,0 +1,434 @@
+{
```

Comment:
<p>Consider loading a dataset and showing a repr here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>